src/FrameBuffer.cpp

```diff
@@ -364,9 +365,18 @@ CachedTexture * FrameBuffer::_getSubTexture(u32 _t)
 	if (y0 + copyHeight > m_pTexture->realHeight)
 		copyHeight = m_pTexture->realHeight - y0;
 
-	glBindFramebuffer(GL_READ_FRAMEBUFFER, m_FBO);
+	if (m_pTexture->frameBufferTexture == CachedTexture::fbMultiSample)
+	{
+		resolveMultisampledTexture(true);
+		glBindFramebuffer(GL_READ_FRAMEBUFFER, m_resolveFBO);
+	}
+	else{
+		glBindFramebuffer(GL_READ_FRAMEBUFFER, m_FBO);
+	}
+
```

Comment:
Yes, it seems to be GLES specific.

GLES
GL_INVALID_OPERATION is generated if **GL_SAMPLE_BUFFERS for the read buffer is greater than zero** and the formats of draw and read buffers are not identical, or **the source and destination rectangles are not defined with the same (X0, Y0) and (X1, Y1) bounds**.

Corresponding condition for OpenGL is much less restrictive:
GL_INVALID_OPERATION is generated if the **value of GL_SAMPLE_BUFFERS for both read and draw buffers is greater than zero** and **the dimensions of the source and destination rectangles is not identical**.

Do you see the difference? We have multi sampled read buffer, the dimensions of the source and destination rectangles are identical, but (X0, Y0) and (X1, Y1) bounds are not the same. Conditions are ok for OpenGL, but not for GLES3.

Let's make it as this:
```
#ifdef GLESX
	if (m_pTexture->frameBufferTexture == CachedTexture::fbMultiSample){
		resolveMultisampledTexture(true);
		glBindFramebuffer(GL_READ_FRAMEBUFFER, m_resolveFBO);
	} else
#endif
	glBindFramebuffer(GL_READ_FRAMEBUFFER, m_FBO);

```